testsuite/bridged_checker/tests/py_ac/ac.py

```diff
@@ -1 +1 @@
-print -1, 1
+print - 1, 1
```

Comment:
Should we be running black on the testsuite? This is it attempting to apply Python 3 formatting on Python 2 code.

Reply:
Yikes... can we configure black somehow? Right now it takes a directory. We can apply to dmoj, but then there are some other files that deserve to be scanned... I'll do some research

Reply:
We have options to install `black[python2]`, but I think that's not the solution here.

Reply:
Definitely shouldn't be formatting the test suite. We could have deliberate syntax errors and other stuff. Should probably just exclude the entire directory from black and flake8.

Reply:
flake8 already excludes it. We can probably just run `black` with `--exclude testsuite`. I'll try that

Reply:
The command is `--exclude testsuite/`, since it seems to be a regex, it will ignore `testsuite.py`

Reply:
But happily, it will be caught by `flake8`.